FORTRAN/H5D/h5_version.h.in

```diff
@@ -0,0 +1,23 @@
+! Version numbers
+!
+! For major interface/format changes
+!
+#define H5_VERS_MAJOR @H5_VERS_MAJOR@
+!
+! For minor interface/format changes
+!
+#define H5_VERS_MINOR @H5_VERS_MINOR@
+!
+! For tweaks, bug-fixes, or development
+!
+#define H5_VERS_RELEASE @H5_VERS_RELEASE@
+
+! macros for comparing versions
+
```

Comment:
Also, how to handle the !defined(H5_USE_XX_API) usage in fortran?

Reply:
Fortran does not have any versioned APIs, so from an application's point of view it should not be needed.

Reply:
I've not checked the examples with an HDF5 version other than the default API mappings.

Reply:
I'll do that afterwards.